박민영/Floyd Warshall (플로이드 워셜)/백준/11404.java

```diff
@@ -0,0 +1,61 @@
+/*
+ * @author Minyeong Park
+ * @date 2023.03.27.
+ * 플로이드
+ * 코드 개선 참고 : https://steady-coding.tistory.com/96, https://bbangson.tistory.com/108
+ * 문제 링크 : https://www.acmicpc.net/problem/11404
+ */
+
+
+import java.io.*;
+import java.util.*;
+
+public class Main {
+    public static void main(String[] args) throws IOException {
+        BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
+        StringTokenizer st;
+        int n = Integer.parseInt(br.readLine());
+        int m = Integer.parseInt(br.readLine());
+
+        int INF = 10000001; // Integer.MAX_VALUE로 설정하면 안 됨! (오버플로우 발생 가능함) // INF는 100000(가중치의 최댓값) * 99 (정점개수는 최대 100개)
+
+        int[][] d = new int[n + 1][n + 1];
+        for (int i = 0; i <= n; i++) {
+            Arrays.fill(d[i], INF);
+        }
+        for (int i = 1; i <= n; i++) {
+            d[i][i] = 0;
+        }
```

Comment:
```suggestion
        int[][] d = new int[n + 1][n + 1];
        for (int i = 0; i <= n; i++) {
            Arrays.fill(d[i], INF);
            d[i][i] = 0;
        }
```

이렇게 하면 반복문을 줄일 수 있을 것 같습니다!

Reply:
헉 오옷 풀 때는 인식하지 못했는데 감사합니다!!